### ISSUE TITLE
[vtk] x64-windows build fails to download vtk

### ISSUE THREAD

**Issue by smoked-herring** [microsoft/vcpkg#14926]
****

**Is your feature request related to a problem? Please describe.**
I'm trying to build a new port so I made a draft PR. It fails to be built on x64-windows because of vtk download failure. Log: https://dev.azure.com/vcpkg/c1ee48cb-0df2-4ab3-8384-b1df5a79fe53/_apis/build/builds/46090/artifacts?artifactName=failure%20logs%20for%20x64-windows&api-version=6.0&%24format=zip

Quote:

```
-- Downloading https://gitlab.kitware.com/vtk/vtk-m/-/archive/f2aa6ad5be1a97e3fb41ef4680ee2c76c3434ac0/vtk-m-f2aa6ad5be1a97e3fb41ef4680ee2c76c3434ac0.tar.gz...
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:105 (message):
  

  File does not have expected hash:

          File path: [ D:/downloads/temp/vtk-vtk-m-f2aa6ad5be1a97e3fb41ef4680ee2c76c3434ac0.tar.gz ]
      Expected hash: [ 2f2a273f74d9a583df9e25a4792440d8d89652fa14b3153f2ea5afbd329b50970e7b9bd68e0ccd036baf5c1f3ad7a8302d95c01dbb30d9a46c045987eebf5370 ]
        Actual hash: [ 35e8a2c0ad6cd3c1f02a71a50d781c89f93909ad27030b406fd69f4fea5c1862c48a6e541fd07562947322c3a69bdfdb54206ae51bb86ef7a710f9e9898e9638 ]

  The file may have been corrupted in transit.  This can be caused by
  proxies.  If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY
  environment variables to
  "https://user:password@your-proxy-ip-address:port/".

Call Stack (most recent call first):
  scripts/cmake/vcpkg_download_distfile.cmake:191 (test_hash)
  scripts/cmake/vcpkg_from_gitlab.cmake:123 (vcpkg_download_distfile)
  ports/vtk-m/portfile.cmake:40 (vcpkg_from_gitlab)
  scripts/ports.cmake:136 (include)
```

**Proposed solution**
Fix downloading vtk.

**Describe alternatives you've considered**
?

**Additional context**
?

**Comment by Hoikas** [comment]
Fixed by #14884

**Comment by strega-nil** [comment]
#14884 fixed this :)
